test/integration/monitoring/serverspec/monitoring_services_spec.rb

```diff
@@ -49,3 +51,9 @@
     expect(port(36638)).to be_listening
   end
 end
+
+describe 'Monitoring services for CentOS 7', if: os[:release].to_i >= 7 do
+  it 'has the kairosdb service running as the kairosdb user' do
+    expect(command('ps h -u kairosdb -o cmd').stdout).to match(/.*org\.kairosdb\.core\.Main.*/)
```

Comment:
Quick note: ServerSpec does not provide a way to check which user runs a service and its `process` directive cannot be used because the process is `java`. Instead, we use this command to list the processes owned by the `kairosdb` user (printing only the command and no headers) and check that the command is actually the KairosDB one.